config.libsonnet

```diff
@@ -43,5 +43,8 @@
     // For links between grafana dashboards, you need to tell us if your grafana
     // servers under some non-root path.
     grafanaPrefix: '',
+
+    // This list of filesystem is referenced in various expressions.
+    fstypeSelector: 'fstype=~"%s"' % std.join('|', ['ext[234]', 'btrfs', 'xfs', 'zfs']),
```

Comment:
I think this array should be a separate field, so people could just add additional filesystems without having to enumerate the default ones again

Reply:
sounds good, you mean like:
```
fstypes = ['ext[234]', 'btrfs', 'xfs', 'zfs'],
fstypeSelector: 'fstype=~"%s"' % std.join('|', fstypes),
```
?

Reply:
along those lines yes (it's gonna be `self.fstypes` in the second line, but yes like that)